eknrenderer/eknrenderer/eknr-renderer.c

```diff
@@ -0,0 +1,1311 @@
+/* Copyright 2018 Endless Mobile, Inc. */
+
+#include <string.h>
+#include <stdlib.h>
+
+#include <endless/endless.h>
```

Comment:
This is the old eos-sdk. Why do you need it?

Reply:
`eos_get_license_display_name`